torch/_higher_order_ops/map.py

```diff
@@ -242,7 +244,7 @@ def trace_map(proxy_mode, func_overload, f, xs, pos_args):
 
     proxy_mode.tracer.root.register_module(next_name, body_graph)
 
-    with disable_proxy_modes_tracing():
+    with disable_proxy_modes_tracing(pre_dispatch=proxy_mode.pre_dispatch):
```

Comment:
I think you removed this in the last PR 🤔 not updated? https://github.com/pytorch/pytorch/pull/115314/files#diff-6960ce90e7162c0953b1ca07e92e7f0f2f6ba63b427b42df593e20cc6a096bb7R1118